steps/python/python.deploy.yml

```diff
@@ -0,0 +1,101 @@
+# python.deploy
+#
+# Deploy Python package to PyPI and optionally to a Docker registry.
+#
+
+parameters:
+  pypiConnector: "testpypi-tomtom-dev"
+  dockerDeploy: true
+  dockerRegistryConnector: "tomtom-docker-registry-bintray"
+  dockerRepo: "python"
+  dockerDeployFile: "Dockerfile"
+
+steps:
+- checkout: self
+  # Allow scripts to access the system token so that the pipeline can push git changes (eg. version bump)
+  persistCredentials: true
+
+- script: |
+    git config --global user.email "44620811+tt-ci@users.noreply.github.com"
```

Comment:
The username and email probably shouldn't be hard-coded

Reply:
Removed it and making use of variable groups to avoid modifying in every single git repo to pass the user/mail combo